templates/template_db.go

```diff
@@ -0,0 +1,128 @@
+package templates_v1
+
+import (
+	"fmt"
+
+	"context"
+
+	"github.com/jackc/pgx"
+)
+
+func dbConnection() (*pgx.Conn, error) {
+	config := pgx.ConnConfig{
+		Host:     "localhost",
+		Database: "tsg",
+		Port:     26257,
+		User:     "root",
+	}
+
+	return pgx.Connect(config)
```

Comment:
Can we move to a connection pool instead?  I have examples of this from `pg_prefaulter` to crib:

https://github.com/joyent/pg_prefaulter/blob/0038505824e2f32b694a7b68e21ce4c693696f27/agent/agent.go#L56-L57
https://github.com/joyent/pg_prefaulter/blob/0038505824e2f32b694a7b68e21ce4c693696f27/agent/db.go#L228-L247
https://github.com/joyent/pg_prefaulter/blob/0038505824e2f32b694a7b68e21ce4c693696f27/config/config.go#L240-L260

?